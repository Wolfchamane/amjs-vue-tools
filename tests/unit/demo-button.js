/* istanbul ignore file */
import DemoButton from '@/components/button';
import renderer   from '#/utils/renderer';

describe('DemoButton - Snapshot', () =>
{
    it('Default', async () => expect(await renderer(DemoButton, null, true)).toMatchSnapshot());
    it('With [type="submit"]', async () => expect(await renderer(DemoButton, { type : 'submit' }, true)).toMatchSnapshot());
});
