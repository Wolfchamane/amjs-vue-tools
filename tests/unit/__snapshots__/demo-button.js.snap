// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DemoButton - Snapshot Default 1`] = `<button type="button"></button>`;

exports[`DemoButton - Snapshot With [type="submit"] 1`] = `<button type="submit"></button>`;
